src/librustc_back/dynamic_lib.rs

```diff
@@ -276,7 +276,8 @@ mod dl {
             None => {
                 let mut handle = ptr::null_mut();
                 let succeeded = unsafe {
-                    GetModuleHandleExW(0 as DWORD, ptr::null(), &mut handle)
+                    let empty_flag = 0;
+                    GetModuleHandleExW(empty_flag, ptr::null(), &mut handle)
```

Comment:
I don't think it is a good practice to denote a bit flag with a plain integer literal `0`. It should be distinguished from a number `0` to which we add `1` or whatever.